x/concentrated-liquidity/incentives.go

```diff
@@ -765,6 +769,10 @@ func (k Keeper) claimAllIncentivesForPosition(ctx sdk.Context, positionId uint64
 		return sdk.Coins{}, sdk.Coins{}, err
 	}
 
+	if err := k.updateUptimeAccumulatorsToNow(ctx, position.PoolId); err != nil {
+		return sdk.Coins{}, sdk.Coins{}, err
+	}
+
```

Comment:
@stackman27 I know you're working on adding and testing this change as well so I'm going to merge mine once CI passes to unblock further work. My understanding is that tests are TBD in https://github.com/osmosis-labs/osmosis/pull/5076

Let me know if this sounds good with you